src/pages/billing/models/BillingAccount.ts

```diff
@@ -0,0 +1,5 @@
+export interface BillingAccount {
```

Comment:
Is it necessary/preferred in typescript to have each of these definitions in their own file?

Reply:
For clarity, can we call this `GoogleBillingAccount`? It took me a while to figure out what it was.

Reply:
to your earlier question, current guidance is to have the type defined within the module that "owns" or emits it... In many cases, this should push types to be near the ajax "data-client" module, or a provider if one exists... But for cases where a number of modules make use of the type and there is no clear "owner", having a shared type be placed further up the directory tree is a good way to go.

Reply:
@msilva-broad sorry, I'm not understanding how to apply your answer to this case. Do you think we should have all these small individual files for each definition?

Reply:
Sorry, but I have to resolve this in order to be able to merge.